locales/en/plugin__odf-console.json

```
@@ -71,10 +71,10 @@
   "Choose your configuration preference to protect resources (application volumes/PVCs, or Kubernetes objects).": "Choose your configuration preference to protect resources (application volumes/PVCs, or Kubernetes objects).",
   "You have selected {{count}} namespaces, to view or change your selection go back to the previous step._one": "You have selected {{count}} namespaces, to view or change your selection go back to the previous step.",
   "You have selected {{count}} namespaces, to view or change your selection go back to the previous step._other": "You have selected {{count}} namespaces, to view or change your selection go back to the previous step.",
-  "Secure namespaces as per recipe definition.": "Secure namespaces as per recipe definition.",
-  "Recipe": "Recipe",
   "Secure selected namespace by defining resource label expressions.": "Secure selected namespace by defining resource label expressions.",
   "Resource label": "Resource label",
+  "Secure namespaces as per recipe definition.": "Secure namespaces as per recipe definition.",
```

assuming `Recipe` is a `kind`...
```suggestion
  "Secure namespaces as per Recipe definition.": "Secure namespaces as per recipe definition.",
```

---

Changes done